rules/defaults.yaml

```diff
@@ -159,6 +159,10 @@
 		Range: 4c0
 	Tooltip:
 		Description: Tech Building
+	CaptureNotification:
+		Notification: TechBuildingCaptured
+	ActorLostNotification:
```

Comment:
This doesn't play at all when the building is captured, but only when it got destroyed.
We might want to expand `CaptureNotification` here to play a sound to the old owner
(if that isn't already possible in some way).